pkg/stanza/operator/input/windows/input_test.go

```diff
@@ -48,6 +48,21 @@ func TestInputStart_LocalSubscriptionError(t *testing.T) {
 	assert.ErrorContains(t, err, "The specified channel could not be found")
 }
 
+// TestInputStart_NoErrorIfIgnoreChannelErrorsEnabled ensures no error is thrown when ignore_channel_errors flag is enabled
+// Other existing tests ensures the default behaviour of error out when any error occurs while subscribing to the channel
+func TestInputStart_NoErrorIfIgnoreChannelErrorEnabled(t *testing.T) {
```

Comment:
Could you please also add a test with the default behavior, the case that we still expect error.